src/tests/unit/handler.test.js

```diff
@@ -61,3 +61,20 @@ describe('Handler throws 405 when httpMethod is not GET', () => {
     })
   ))
 })
+
+describe('Handler throws error when called with queryStringParameters', () => {
+  it('verifies response is error 400 and has Bad Request problem response', async function () {
+    const queryStringParameters = 'sample.query.string.parameter'
+    const event = { path: '/journal', httpMethod: 'GET', queryStringParameters: queryStringParameters }
+    const response = await handler(event)
+    expect(response.headers).to.have.property('Content-Type')
+    expect(response.headers['Content-Type']).to.equal('application/problem+json')
+    expect(response.statusCode).to.equal(400)
```

Comment:
In the future we will have many bad request responses. We should not duplicate these assertions.I think the assertions  about the headers  and should be in another more general test. I am not sure though. I would suggest a quick chat with Rurik

Reply:
Moved testing of headers to own test, this testing might be extended i later PR's